internal/controller/api/v1alpha1/host_types.go

```diff
@@ -17,25 +17,117 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/openshift/assisted-service/models"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+type HostMemory struct {
+	PhysicalBytes int64 `json:"physicalBytes,omitempty"`
+	UsableBytes   int64 `json:"usableBytes,omitempty"`
+}
+
+type HostCPU struct {
+	Count        int64    `json:"count,omitempty"`
+	Frequency    int64    `json:"frequency,omitempty"`
```

Comment:
this was originally type `float64`, but k8s API does not support that:

```golang
controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role paths="./..." output:rbac:dir=internal/controller/config/rbac \
	webhook paths="./..." output:crd:artifacts:config=internal/controller/config/crd/bases
/home/nmagnezi/go/src/github.com/assisted-service/models/cpu.go:28:12: unsupported type "float64"
Error: not all generators ran successfully
```

let me know what should be used as `type` here.

Reply:
thanks!